src/components.d.ts

```diff
@@ -11,7 +11,7 @@ import { address as address1 } from "./components/address-display/index";
 import { Color, Data, Fill, Icon, Message, Notice, Option, Trigger } from "./model";
 import { Button } from "./components/Button";
 import { CountryCode, Currency, Date, DateRange, DateTime, isoly } from "isoly";
-import { Direction, Type } from "tidily";
+import { tidily, Type } from "tidily";
```

Comment:
Remove Type import.

Reply:
this file is automatically generated and decides on its own imports.

Reply:
fixing this would require a change in all files that imports tidily. i guess skip for now?